[2023-06-04T10:58:26.487+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-04T10:58:24.868568+00:00 [queued]>
[2023-06-04T10:58:26.494+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-04T10:58:24.868568+00:00 [queued]>
[2023-06-04T10:58:26.495+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-04T10:58:26.504+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): scrape_most_popular_titles> on 2023-06-04 10:58:24.868568+00:00
[2023-06-04T10:58:26.509+0000] {standard_task_runner.py:57} INFO - Started process 3124 to run task
[2023-06-04T10:58:26.512+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'most_popular_movies', 'scrape_most_popular_titles', 'manual__2023-06-04T10:58:24.868568+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/most_popular_movies.py', '--cfg-path', '/tmp/tmp99fwrrmh']
[2023-06-04T10:58:26.512+0000] {standard_task_runner.py:85} INFO - Job 91: Subtask scrape_most_popular_titles
[2023-06-04T10:58:26.546+0000] {task_command.py:410} INFO - Running <TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-04T10:58:24.868568+00:00 [running]> on host 7633a4fabdb3
[2023-06-04T10:58:26.610+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='most_popular_movies' AIRFLOW_CTX_TASK_ID='scrape_most_popular_titles' AIRFLOW_CTX_EXECUTION_DATE='2023-06-04T10:58:24.868568+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-04T10:58:24.868568+00:00'
[2023-06-04T10:58:27.452+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/most_popular_movies.py", line 24, in scrape_most_popular_titles
    movie_dict = _scrape_movies(soup)
  File "/opt/airflow/dags/helper/scrape_imdb_charts.py", line 78, in _scrape_movies
    movie_ratings.append(float(rating.find("strong").text))
AttributeError: 'NoneType' object has no attribute 'text'
[2023-06-04T10:58:27.459+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=most_popular_movies, task_id=scrape_most_popular_titles, execution_date=20230604T105824, start_date=20230604T105826, end_date=20230604T105827
[2023-06-04T10:58:27.469+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 91 for task scrape_most_popular_titles ('NoneType' object has no attribute 'text'; 3124)
[2023-06-04T10:58:27.489+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-04T10:58:27.506+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
